src/Schema/AbstractSchemaManager.php

```diff
@@ -177,12 +177,13 @@ public function tableExists(string $tableName): bool
     /**
      * Returns a list of all tables in the current database.
      *
-     * @return array<int, non-empty-string>
+     * @return list<non-empty-string>
      *
      * @throws Exception
      */
     public function listTableNames(): array
     {
+        /** @phpstan-ignore return.type */
```

Comment:
(why) is this necessary? I've tried removing it and running PHPStan locally, and it passed.